docs/src/user-guide/getting-started.md

```diff
@@ -28,14 +28,15 @@ If you want to use a specific shareable config that is hosted on npm, you can us
 
 ```shell
 # use `eslint-config-semistandard` shared config
```

Comment:
```suggestion
# use `eslint-config-semistandard` shared config

```

As this comment applies to all example commands, might be better to insert a blank line here.